build.psm1

```diff
@@ -2245,7 +2260,7 @@ function New-AppxPackage
         # Source Path to the Binplaced Files
         [Parameter(Mandatory = $true)]
         [ValidateNotNullOrEmpty()]
-        [string] $SourcePath,
+        [string] $PackageSourcePath,
```

Comment:
Renamed this parameter to PackageSourcePath for consistency across packing functions